dll/win32/msgina/gui.c

```diff
@@ -210,7 +210,7 @@ StatusDialogProc(
             DlgData_LoadBitmaps(pDlgData);
             if (pDlgData->hBarBitmap)
             {
-                pDlgData->TimerID = SetTimer(hwndDlg, -1, 20, NULL);
+                pDlgData->TimerID = SetTimer(hwndDlg, -1, 200, NULL);
```

Comment:
Please don't change stuff that actually do not bring *improvements* at all. The elapse time parameter is set to this value to nearly match the speed of the bar movement rotation in Windows Server 2003's MSGINA.

Reply:
Noted. Meanwhile, I am interested in slowing down as a test.